accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/impl/AcInstallationServiceImpl.java

```diff
@@ -207,22 +222,38 @@ public InstallationLog apply(String configurationRootPath, String[] restrictedTo
         return installLog;
     }
 
-    /** Common entry point for JMX and install hook, 
-     * TODO: should not be exported as using non-API class PersistableInstllationLogger */
```

Comment:
This TODO should be kept!

Reply:
@kwin please add an issue here (TODOs get forgotten, issues are better in that respect)

Reply:
There is already an issue: https://github.com/Netcentric/accesscontroltool/issues/394.
Todo's don't hurt though because they have a better context!